docs/worker-node/install-cvmfs.md

```diff
@@ -102,14 +102,14 @@ CVMFS uses automount, and the steps to configure it are different on EL6 vs EL7.
 
 Create or edit `/etc/cvmfs/default.local`, a file that controls the
 CVMFS configuration. Below is a sample configuration, but please note
-that you will need to **edit the parts in %RED%red%ENDCOLOR%**. In
+that you will need to **edit the parts in angle brackets**. In
 particular, the `CVMFS_HTTP_PROXY` line below must be edited for your
 site.
 
 ```
 CVMFS_REPOSITORIES="`echo $((echo oasis.opensciencegrid.org;echo cms.cern.ch;ls /cvmfs)|sort -u)|tr ' ' ,`"
-CVMFS_QUOTA_LIMIT=%RED%20000%ENDCOLOR%
-CVMFS_HTTP_PROXY=%RED%"http://squid.example.com:3128"%ENDCOLOR%
+CVMFS_QUOTA_LIMIT=<QUOTA LIMIT>
+CVMFS_HTTP_PROXY="<SQUID URL>:<SQUID PORT>"
```

Comment:
I don't like this, it's not as helpful as having an example that's closer to the real thing.  If we truly can't fix the red color, then I'd rather have the bold statement say that you will need to edit at least the last line.  The 20000 is described below as a reasonable default.

Reply:
Example config files should clearly highlight which fields need to be adjusted. Examples of appropriate values belong in the description of the fields below.

Reply:
Well then clear examples immediately following should probably be added.